cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/AppFabricServer.java

```diff
@@ -147,27 +162,52 @@ protected void startUp() throws Exception {
       )
     ).get();
 
+    int serverPort;
+    if (sslEnabled) {
+      serverPort = cConf.getInt(Constants.AppFabric.SERVER_SSL_PORT);
+      String password = sConf.get(Constants.Security.SSL.KEYSTORE_PASSWORD, generateRandomPassword());
```

Comment:
What is this password for? Why it needs to be configurable since we only use it over the generated cert store?